test/fixtures.yml

```diff
@@ -331,6 +331,12 @@ firstdata_e4:
   login: SD8821-67
   password: T6bxSywbcccbJ19eDXNIGaCDOBg1W7T8
 
+firstdata_e4_v27:
```

Comment:
If these creds are legit and it's okay to have them, we should probably stick the `# Working credentials, no need to replace` comment here; otherwise, we should probably nuke 'em.